KEDamage.cs

```diff
@@ -8,58 +8,94 @@ class KEDamage:Damage
     {
         public struct KineticData
         {
+            /// <summary>
+            /// The pierce of the shot
+            /// </summary>
             public float Pierce;
-
+            /// <summary>
+            /// Distance between the firing unit and target unit
+            /// </summary>
             public float Distance;
-
-            public float ArmorKEFactor;
-
-            public float HealthKEFactor;
-
+            /// <summary>
+            /// Multiplier for armor degradation
+            /// </summary>
+            public float Degradation;
+            /// <summary>
+            /// Multiplier for health damage
+            /// </summary>
+            public float HealthDamageFactor;
+            /// <summary>
+            /// Air friction constant used in calculation of attenuation
+            /// </summary>
             public float FrictionFactor;
```

Comment:
Since you now have it commented and the code better structured, do you need `Factor` at the end of the variable names? Especially since you commented that the air friction is a constant.